providers/oauth/oauth.identity.js

```diff
@@ -29,12 +29,19 @@ ChromeIdentityAuth.prototype.initiateOAuth = function(redirectURIs, continuation
   return false;
 };
 
-ChromeIdentityAuth.prototype.launchAuthFlow = function(authUrl, stateObj, continuation) {
+ChromeIdentityAuth.prototype.launchAuthFlow = function(authUrl, stateObj, interactive, continuation) {
+  if (interactive === undefined) {
```

Comment:
As in the other repo, I think it may be better to go with `typeof interactive === "undefined"` (typeof is guaranteed to return a string, while maybe some JavaScript implementations will be antsy about directly examining something that is undefined - though given that it's a function parameter it may be fine, but still I think it's good to stick w/typeof as a matter of style/habit).